docs/detections/prebuilt-rules/downloadable-packages/0-16-1/prebuilt-rule-0-16-1-aws-efs-file-system-or-mount-deleted.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[prebuilt-rule-0-16-1-aws-efs-file-system-or-mount-deleted]]
+=== AWS EFS File System or Mount Deleted
+
+Detects when a EFS File System or Mount is deleted. An adversary could break any file system using the mount target that is being deleted, which might disrupt instances or applications using those mounts. The mount must be deleted prior to deleting the File System, or the adversary will be unable to delete the File System.
```

Comment:
Change "a EFS" to "an EFS". Also, should file system and mount be capitalized in the first sentence? Only asking because they're not in the ones following--except for in the last sentence.